File: org.kbinani.media/MidiReceivedEventHandler.cs
/*
 * BEventHandler.cs
 * Copyright © 2009-2011 kbinani
 *
 * This file is part of org.kbinani.
 *
 * org.kbinani is free software; you can redistribute it and/or
 * modify it under the terms of the BSD License.
 *
 * org.kbinani is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 */
#if JAVA
//INCLUDE ../BuildJavaUI/src/org/kbinani/media/MidiReceivedEventHandler.java
#else

namespace org.kbinani.media
{

    public delegate void MidiReceivedEventHandler( Object sender, org.kbinani.javax.sound.midi.MidiMessage message );

}
#endif

File: cadencii.media/MidiReceivedEventHandler.java
package com.github.cadencii.media;

import com.github.cadencii.BEventHandler;

public class MidiReceivedEventHandler extends BEventHandler{
    public MidiReceivedEventHandler( Object invoker, String method_name ){
        super( invoker, method_name, Void.TYPE, Object.class, javax.sound.midi.MidiMessage.class );
    }

    public MidiReceivedEventHandler( Class<?> invoker, String method_name ){
        super( invoker, method_name, Void.TYPE, Object.class, javax.sound.midi.MidiMessage.class );
    }
}
